File: U4/src/EJ5.java
import java.util.Scanner;

class EJ5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("introduce la base");
        int base = sc.nextInt();

        triangulo(base);
    }
    static void triangulo (int base){
        for (int c = base ; c>0;c--){
            for (int d = 0; d<c;d++) {
                System.out.print("Q");
            }

            System.out.println();

        }


    }

}

File: KOTLIN U7/untitled/src/Ej5.kt
import java.util.Scanner

fun main() {
    val Scanner = Scanner(System.`in`)
    print("Introduce un numero, sera la longitud del array : ")
    val num = Scanner.nextInt()
    var array = IntArray(num)
    println(" " )

    for (i in 1..num){
        print("Introduce un numero, quedan ${num-i} espacios : ")
        var num2 = Scanner.nextInt()
        array[i-1] = num2
    }
    array.sort()
    print("Array completo : ${array.contentToString()} " )
    println(" ")

    println("El segundo numero mas grande es ${array[array.size - 2]}")

}